### PR TITLE
 Update spring-boot-bom to 1.5.13.Beta1

### PR DESCRIPTION
**Pull request by gytis**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -20,18 +20,18 @@
   <parent>
     <groupId>io.openshift</groupId>
     <artifactId>booster-parent</artifactId>
-    <version>21</version>
+    <version>22</version>
```
Comment (geoand) [inline pom.xml:23]: This seems to have updated some Arquillian version. Do we need to do the same or do we just reuse the deps from the parent? I am assuming it's the latter, but just wanted to double check with you

Comment (metacosm) [reply pom.xml:23]: Yes, I'm worried about the implications as well. Do we have a test run to support that change?

Comment (geoand) [reply pom.xml:23]: I'm believe running any of the booster tests would do the job

Comment (gytis) [reply pom.xml:23]: I only ran the script from booster-common. I could run integration tests on a couple of boosters too

Comment (gytis) [reply pom.xml:23]: Just ran IT on HTTP and CB boosters and all works fine

**Review by geoand — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (gytis) [reply pom.xml:23]: Health check works fine too

**Review by metacosm — APPROVED** [review]

**Pull request merged**